docs/source/2018Michael.rst

```diff
@@ -116,11 +138,33 @@ Hurricane Track and Intensity
     * The maximum surface wind speed at the vortex center in GFS.v16.0.10 reaches to 90 kts, which is 50 kts lower than Best Track data.
     * The minimum sea level pressure from GFS.v16.0.10 is 950 hPa, versus 919 hPa from Best Track data. 
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2018Michael/tracker_Michael_srwv1.png
+      :width: 400
+      :align: center
+
+      Hurricane tracks from SRW_GFSv15p2 (purple dash line), MRW_GFSv16beta (blue line), MRW_GFSv15p2 (red line),  and Best Track (black line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
+
+    * Hurricane track from SRW_GFSv15p2 is closer to the Best Track compared with MRW_GFSv15p2 and MRW_GFSv16beta. 
+
+
+    .. figure:: images/2018Michael/tracker_timeseries_Michael_srwv1.png
+      :width: 1200
+      :align: center
+
+      Time series of the vortex maximum surface wind speed (WS, left panel) and minimum mean sea level pressure (MSLP, right panel)
+
+    * The maximum surface wind speed at the vortex center in SRW_GFSv15p2 (90 kts) does not reach the peak intensity identified in Best Track data (140 kts).
+    * The minimum sea level pressure from SRW_GFSv15p2 948 hpa, which is larger than Best Track data.
+  
+
 ......................
 Summary and Discussion
 ......................
 
 The physics compset of GFS.v16beta in MRW.v1.0 does not show improvements compared with the compset of GFS.v15p2 in simulating Hurricane Michael track and intensity. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.
+The results from SRWapp v1.0 has better track and intensity in simulating Hurricane Michael comparing with the results from MRW app v1.0; however, the SRWapp v1.0 has a newer model and shorter forcasts.
```

Comment:
Linlin, could you please expand the explanation of the meaning of 'a newer model'? 
I like that you combined the results of MRW and SRW in the plot. But I only see SRW_GFSv15p2, but not for SRW_RRFSv1alpha. Is this on purpose?

Reply:
RRFSv1alpha crashed, the old data seem don't work with RRFSv1alpha.

Reply:
I see. Have you tried to reduce the time steps?

Reply:
Yes， I guess the problem is beyond the time step.

Reply:
Thank you for the information Linlin! If so, I think it may worth mentioning here why RRFSv1alpha results are not mentioned here.

Reply:
Yes, thanks!